packages/back-end/src/controllers/experiments.ts

```diff
@@ -531,6 +534,21 @@ export async function postExperiment(
 
   if (requiresWebhook) {
     await queueWebhook(req.organization.id);
+
+    const apiKeys = await getAllApiKeysByOrganization(req.organization.id);
```

Comment:
I would move this logic into the `queueCDNInvalidate` function.  So you just pass in the organization id (and experiment implementation type) and that function takes care of looping through api keys, etc.